src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+    private final DtoResponseMapper<UserResponseDto, User> mapper;
+
+    public AuthenticationController(AuthenticationService authenticationService,
+                                    DtoResponseMapper<UserResponseDto, User> mapper) {
+        this.authenticationService = authenticationService;
+        this.mapper = mapper;
+    }
+
+    @PostMapping("/register")
+    UserResponseDto register(@RequestBody UserRequestDto userRequestDto) {
+
```

Comment:
```suggestion
    UserResponseDto register(@RequestBody UserRequestDto userRequestDto) {
```
Don't you forget to add 'Valid' annotation here ?